AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -2,24 +2,21 @@ import { DropdownStepView, InfoStepView, StepFlow, StepFlowManager, CheckStepVie
 import { checkKuduAvailabilityAsync, checkVnetIntegrationV2Async, checkDnsSettingV2Async, checkAppSettingsAsync, extractHostPortFromConnectionString, extractHostPortFromKeyVaultReference } from './flowMisc.js';
 import { VnetIntegrationConfigChecker } from './vnetIntegrationConfigChecker.js';
 import { VnetDnsWordings } from './vnetDnsWordings.js';
-import { CommonWordings } from './commonWordings.js';
 
 export class ConnectionStringType {
     static get StorageAccount() { return 'StorageAccount' };
+    static get BlobStorageAccount() { return 'BlobStorageAccount' };
+    static get QueueStorageAccount() { return 'QueueStorageAccount' };
+    static get FileShareStorageAccount() { return 'FileShareStorageAccount' };
     static get ServiceBus() { return 'ServiceBus' };
-    static get EventHubs() { return 'EventHubs' };
+    static get EventHubs() { return 'EventHubs' };       
 }
 
 export var functionsFlow = {
     title: "Connectivity issues",
     async func(siteInfo, diagProvider, flowMgr) {
 
-        var isKuduAccessiblePromise = null;

```

Comment:
Why are we modifying this logic?

Reply:
While check in my latest code to Git , I have missed the changes from yifguoMSFT (on Jan 11) . I will take the latest code from Main branch and do check in again.